bookwyrm/templates/ostatus/remote_follow_button.html

```diff
@@ -0,0 +1,12 @@
+{% load i18n %}
+{% if request.user == user %}
+{% else %}
+
+<div class="field mb-0">
+    <div class="control">
+        <a class="button is-small is-link" href="{% url 'remote-follow-page' %}?user={{ user.username }}" target="_blank" rel="noopener noreferrer" onclick="BookWyrm.displayPopUp(`{% url 'remote-follow-page' %}?user={{ user.username }}`, `remoteFollow`); return false">
```

Comment:
I saw in the issue that you weren't sure as to how to make the pop-up window accessible: I suggest adding an ARIA annotation, saying that the link opens in a pop-up window. From that point on, the APIs from the OS take over and describe the new (focused) window.

Why `aria-description`?

- It is [pretty well supported](https://a11ysupport.io/tech/aria/aria-description_attribute), except on iOS)
- `aria-label` override the text content of the link, which is not good in this situation
- `title` has horrible screen reader support
- `is-sr-only` could be used in the button's text content, but I think it would insert whitespace and mess up the button's dimensions

```suggestion
        <a class="button is-small is-link" href="{% url 'remote-follow-page' %}?user={{ user.username }}" target="_blank" rel="noopener noreferrer" onclick="BookWyrm.displayPopUp(`{% url 'remote-follow-page' %}?user={{ user.username }}`, `remoteFollow`); return false" aria-description="{% trans "This link opens in a pop-up window" %}">
```

Or, we could add the notice directly in the button text, without hiding it. The button would be bigger, the interface could appear messier, but it could be better for some users.

An alternative would be to have the text visible, outside the button but visually linked to it (close enough that we know it's that link that opens in a new window), with an `aria-describedby` on the link pointing to that text.

Reply:
Thank you! Is this a best practice for any link with a `target="_blank"`, or just for popups specifically?

Reply:
Thanks @joachimesque!

Generally speaking I'm not in favour of pop-ups, this is one of those rare exceptions where I think it is genuinely the smoothest option for most users. Still, it's confusing if you are blocking pop-ups. I think I actually prefer the last option. What do you think about this, @joachimesque @mouse-reeve ? 

![pop-up](https://user-images.githubusercontent.com/17669239/143784633-659934b8-3695-422e-b7ec-ad3e42dbc3df.png)

If we go with it, is it best to use only `aria-describedby` or is it helpful to include both `aria-describedby` _and_ `aria-description`? Or is that actually annoying/confusing for people using screen-readers?

Reply:
The last option is IMO the best scenario: the information is available to all users. `aria-description` won't be  useful in this case, you can only use `aria-describedby` :)

Reply:
I like that option as well